tools/fluent/basic_syntax.md

```diff
@@ -0,0 +1,131 @@
+* Attribute definitions must be indented and start with a period.
+* There is an equal sign following the message identifier, even if there is no value (i.e. the value is *Null*).
+
+```PROPERTIES
+instructions-link = Log out
+    .tooltip = Disconnect from this account
+```
+
+In this case, `instructions-link` has a *value* (`Log out`) and a `tooltip` *attribute* (`Disconnect from this account`).
+
+It’s important to note that, in FTL files, **indendation** is part of the syntax; indenting elements incorrectly will lead to parsing errors.
+
+## Selectors and plurals
+
+With the select expression, a single message can provide several alternatives. The selected value will depend on the value of an external variable or function:
+
+```PROPERTIES
+welcome-pet = { $petFamily ->
+        [canine] Hello dog
+        [feline] Hello cat
+        *[other] Hello pet
+    }
+```
+
+Notice that both the variants and the closing curly brace are indented. The same string can also be written as:
+
+```PROPERTIES
+welcome-pet =
+    { $petFamily ->
+        [canine] Hello dog
+        [feline] Hello cat
+        *[other] Hello pet
+    }
+```
+
+The message displayed by the UI will change based on the value of the external variable `$petFamily`. Notice that one of the variants starts with a `*`: that indicates the **default option**, and it must always be defined in a selector.
+
+The most common use of selectors is for plural forms:
+
+```PROPERTIES
+emails = { $unreadEmails ->
+        [one] You have one unread email.
+       *[other] You have { $unreadEmails } unread emails.
+    }
+```
+
+In this case, the message displayed will change based on the numeric value of `$unreadEmails`. For plurals, the variant key can either be a perfect match to a number or one of the [CLDR plural categories](http://www.unicode.org/cldr/charts/30/supplemental/language_plural_rules.html). Note how this allows to define special cases, beyond the number of plurals expected for a language:
+
+```PROPERTIES
+emails = { $unreadEmails ->
+        [one] You have one unread email.
+        [42] You have { $unreadEmails } unread emails. So Long, and Thanks for All the Fish.
+       *[other] You have { $unreadEmails } unread emails.
+    }
+```
+
+## Variants
+
+While in most cases messages will have a single value, it’s also possible to define multiple **variants**: different values, each one associated to a key, that can be selected when referencing the message ID from another message. Unlike selectors, the message doesn’t depend on the value of an external variable or function.
+
+This can be useful, for example, to decline names according to grammatical case.
+
+```PROPERTIES
+-brand-name = {
+       *[nominative] Aurora
+        [genitive] Aurore
+        [dative] Aurori
+        [accusative] Auroro
+        [locative] Aurori
+        [instrumental] Auroro
+    }
+
+about = O { -brand-name[locative] }
+```
+
+As for selectors, there must be a default variant, identified by `*`.
```

Comment:
IMHO, we should also add how to use gender of brand names here, and in brand_names.md.

Reply:
I disagree, it would be extremely confusing. That's also why I was asking for an alternative example.

I'm happy to have examples for both case and gender, but it should not be about brand names.

We know that we don't want people to localize brand, why give them examples of exactly what we don't want them to do? If we manage to change the policy, that definitely needs to be added to into brand_names.md

Reply:
Genders are useful to accord verbs in the past tense or the past participle. I don't think we should add genders right here, but I'd like to see an example in `brand_names.md`.

Reply:
Can you think of an example of string with gender variants that doesn't imply declining the brand name?

Reply:
IIRC, Polish had `program brandshortname` in update dialogs only to adjust for the verb in the sentence.